src/Impl/GuzzleHttpRequester.php

```diff
@@ -102,7 +103,7 @@ private function parseFailureResponse(ResponseInterface $response): FailureRespo
      */
     private function parseResponseToJsonObjects(ResponseInterface $response): array
     {
-        $singleResponseJsons = self::splitToSingleJsons((string)$response->getBody());
+        $singleResponseJsons = MultipleJsonsSplitter::splitToSingleJsons((string)$response->getBody());
```

Comment:
Extracted to own class and added handling of edge cases:
 - a `{` inside json value
 - a `\"` inside json value